addons/block_code/ui/blocks/utilities/parameter_input/parameter_input.gd

```diff
@@ -11,19 +11,72 @@ signal text_modified
 
 @export var variant_type: Variant.Type = TYPE_STRING
 @export var block_type: Types.BlockType = Types.BlockType.VALUE
+var option: bool = false
 
 var block: Block
 
-@onready var _line_edit := %LineEdit
+@onready var _panel := %Panel
 @onready var snap_point := %SnapPoint
+@onready var _input_switcher := %InputSwitcher
 
+## Inputs
+# Text
+@onready var _text_input := %TextInput
+@onready var _line_edit := %LineEdit
+# Color
+@onready var _color_input := %ColorInput
+# Option Dropdown
+@onready var _option_input := %OptionInput
+# Vector2
+@onready var _vector2_input := %Vector2Input
+@onready var _x_line_edit := %XLineEdit
+@onready var _y_line_edit := %YLineEdit
+# Bool
+@onready var _bool_input := %BoolInput
+@onready var _bool_input_option := %BoolInputOption
+
+
+func set_raw_input(raw_input):
+	if option:
+		_panel.visible = false
```

Comment:
You can save some indentation by returning earlier:
```
	if not option:
		return

	_panel.visible = false
	(...)

```

Reply:
The diff looks kind of weird, but I can't return early because if `option` is false I want to do the normal `variant_type` logic.